src/app/organizations/manage/events.component.ts

```diff
@@ -120,4 +142,11 @@ export class EventsComponent implements OnInit {
         this.morePromise = null;
         this.refreshPromise = null;
     }
+
+    private appApiPromiseUnfulfilled() {
+        if (this.refreshPromise != null || this.morePromise != null || this.exportPromise != null) {
```

Comment:
Couldn't you just return this value directly instead of using the `if` block? Unless this is for readability or following a pattern

Reply:
nice, yep :)